public/app/core/components/AppChrome/QuickAdd/QuickAdd.tsx

```diff
@@ -41,16 +41,13 @@ export const QuickAdd = ({}: Props) => {
   return createActions.length > 0 ? (
     <>
       <Dropdown overlay={MenuActions} placement="bottom-end">
-        {isSmallScreen ? (
-          <ToolbarButton iconOnly icon="plus-circle" aria-label="New" />
-        ) : (
-          <Button variant="primary" size="sm" icon="plus">
-            <div className={styles.buttonContent}>
-              <span className={styles.buttonText}>New</span>
-              <Icon name="angle-down" />
-            </div>
-          </Button>
-        )}
+        {(isOpen) =>
+          isSmallScreen ? (
+            <ToolbarButton iconOnly icon="plus-circle" aria-label="New" />
+          ) : (
+            <ToolbarButton iconOnly icon="plus" isOpen={isOpen} aria-label="New" />
```

Comment:
hmmm it matches what we do with `Timepicker` tho, trying to avoid inconsistencies 😅 

![image](https://user-images.githubusercontent.com/20999846/202246289-43bb4b84-9b89-47aa-aad2-222e633ef12b.png)

another (smaller) problem is the `aria-expanded` state in `ToolbarButton` is keyed off `isOpen`.

idk, wdyt? 🤔

Reply:
feels like another good design system topic tbh. github is pretty consistent about never rotating the caret, whereas we pretty consistently do. don't think either is wrong, just a matter of preference and deciding on a convention 🤓

Reply:
@ashharrison90 yea, true. I am fine either way :) Don't think it's something users will notice / think about either way